app/channels/application_cable/connection.rb

```diff
@@ -4,15 +4,13 @@ class Connection < ActionCable::Connection::Base
     def connect
       self.current_user = find_verified_user
     end
-    
+
     def find_verified_user
-      if cookies.signed['user_token'] != nil
-        user = User.where(persistence_token: cookies.signed['user_token'])
-        if user.any?
-          return user.first
-        else
-          return nil
-        end
+      return nil if cookies.signed['user_token'].nil
```

Comment:
This will now return nil without executing the rest of the method if `cookies.signed['user_token'].nil` is true. Did you mean `unless` rather than `return`?